packages/amplify_datastore/ios/Classes/DataStoreHubEventStreamHandler.swift

```diff
@@ -0,0 +1,133 @@
+                        do {
+                            let syncQueriesReady =  try FlutterSyncQueriesReadyEvent(payload: payload)
+                            self.sendEvent(flutterEvent: syncQueriesReady.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: "syncQueriesReady")
+                        }
+                    case HubPayload.EventName.DataStore.ready :
+                        do {
+                            let ready =  try FlutterReadyEvent(payload: payload)
+                            self.sendEvent(flutterEvent: ready.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: "ready")
+                        }
+                    case HubPayload.EventName.DataStore.outboxMutationEnqueued :
+                        do {
+                            guard let outboxMutationEnqueued = payload.data as? OutboxMutationEvent else {
+                                      throw FlutterDataStoreError.hubEventCast
+                                  }
+                            let modelName = outboxMutationEnqueued.modelName
+                            let schema = try FlutterDataStoreRequestUtils.getModelSchema(modelSchemas: self.flutterModelRegistration?.modelSchemas ?? [:], modelName: modelName)
+
+                            let flutterOutboxMutationEnqueued = try FlutterOutboxMutationEnqueuedEvent(outboxMutationEnqueued: outboxMutationEnqueued, eventName: payload.eventName, schema: schema)
+                            self.sendEvent(flutterEvent: flutterOutboxMutationEnqueued.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: payload.eventName)
+                        }
+                    case HubPayload.EventName.DataStore.outboxMutationProcessed :
+                        do {
+                            guard let outboxMutationProcessed = payload.data as? OutboxMutationEvent else {
+                                      throw FlutterDataStoreError.hubEventCast
+                                  }
+                            let modelName = outboxMutationProcessed.modelName
+                            let schema = try FlutterDataStoreRequestUtils.getModelSchema(modelSchemas: self.flutterModelRegistration?.modelSchemas ?? [:], modelName: modelName)
+
+                            let flutterOutboxMutationProcessed = try FlutterOutboxMutationProcessedEvent(outboxMutationProcessed: outboxMutationProcessed, eventName: payload.eventName, schema: schema)
+                            self.sendEvent(flutterEvent: flutterOutboxMutationProcessed.toValueMap())
+                        } catch {
+                            self.sendError(description: castingError, eventName: payload.eventName)
+                        }
+                    default:
+                        print("d")
```

Comment:
hmmm

Reply:
lol - Praveen "d" stands for default, it totally makes sense.

Reply:
Fixed.